.vscode/settings.json

```diff
@@ -1,13 +1,37 @@
 // Place your settings in this file to overwrite default and user settings.
 {
-  "editor.tabSize": 2,
+  "debug.internalConsoleOptions": "openOnSessionStart",
+  "editor.codeActionsOnSave": {
+    "editor.action.fixAll": true,
+    "source.fixAll.eslint": true,
+    "source.fixAll.markdownlint": true
+  },
   "editor.insertSpaces": true,
-  "editor.formatOnSave": true,
-  "search.exclude": {
-    "**/node_modules": true,
-    "**/bower_components": true,
-    "**/.nyc_output": true,
-    "**/.rush": true
+  "editor.tabSize": 2,
+  "eslint.enable": true,
+  "eslint.format.enable": true,
+  "eslint.lintTask.enable": true,
+  "eslint.options": {
+    "overrideConfigFile": "./package.json",
+    "resolvePluginsRelativeTo": "../../common/temp/node_modules/.pnpm/@itwin+eslint-plugin*/node_modules",
```

Comment:
temporary solution until we adopt eslint flat configs & use the rushstack/eslint-plugin-patch

(basically need to reapply this [commit](https://github.com/iTwin/itwinjs-core/pull/5460/commits/a2b2e6c98d466d4cd8e7384f0f13218fa13f073d) later on)

fyi @Jake-Screen @paulius-valiunas